src/signing/interaction.ts

```diff
@@ -0,0 +1,40 @@
+import { print } from "src/common/utils"
+import { getMsgFromUser, interactionChoiceQuestions, rawSignParamsQuestions } from "./utils"
+import inquirer from "inquirer"
+import { readFileSync } from "fs"
+import Entropy from "@entropyxyz/sdk"
+import { EntropySign } from "./main"
+
+export async function entropySign (entropy: Entropy, endpoint: string) {
+  const Sign = new EntropySign(entropy, endpoint)
+  const { interactionChoice } = await inquirer.prompt(interactionChoiceQuestions)
+  switch (interactionChoice) {
+  case 'Raw Sign': {
+    const { msg, msgPath } = await getMsgFromUser(inquirer)
+    const { hashingAlgorithm, auxiliaryDataFile } = await inquirer.prompt(rawSignParamsQuestions)
+    let hash = hashingAlgorithm
+    const auxiliaryData = JSON.parse(readFileSync(auxiliaryDataFile).toString())
```

Comment:
I think this is the same?

```suggestion
    const auxiliaryData = JSON.parse(readFileSync(auxiliaryDataFile, 'utf-8'))
```